cli/pkg/commands/images/images.go

```diff
@@ -103,6 +104,25 @@ func (m imagesModel) buildModelImage() func() tea.Msg {
 	}
 }
 
+func (m imagesModel) describeModelImage() func() tea.Msg {
+	c := m.edgeClient
+	var modelImage modelImageDescribeMsg
+	return func() tea.Msg {
+		models, err := c.GetModelImages()
+		if err != nil {
+			return common.ErrMsg{err}
+		}
+
+		for _, model := range models {
+			if model.ModelID == m.flags[flags.FlagModelID.String()] && model.Version == m.flags[flags.FlagVersionName.String()] {
+				modelImage.selectedImage = model
+			}
```

Comment:
There can be more than one artifact, right? At least based on this:
https://github.com/opendatahub-io/ai-edge/blob/aa734d2d1e2ab789baa62e198d9eec93d000319f/cli/pkg/edgeclient/client.go#L139-L155

In that case we will be showing just the first image/artifact found. Currently we add one artifact with the same name as the model. Or for now we count with just one artifact? If so, we can at least track it in JIRA.

Reply:
Right now it should be only one artifact per version. So it's safe to assume that it's only a single artifact.